analysis/shaman/src/ElementalBlast.tsx

```diff
@@ -60,35 +38,38 @@ class ElementalBlast extends Analyzer {
     );
   }
 
-  get elementalBlastUptime() {
-    return this.resultDuration / this.owner.fightDuration;
+  get totalUptime() {
+    return this.critUptime + this.hasteUptime + this.masteryUptime;
```

Comment:
this doesnt work

Reply:
you can have/will have multiple buffs at the same time so you cant just add the uptimes